Makefile

```diff
@@ -82,6 +82,14 @@ ifeq ($(DISABLE_CGO), 1)
 	override BUILDTAGS = exclude_graphdriver_devicemapper exclude_graphdriver_btrfs containers_image_openpgp
 endif
 
+ARCHES=amd64 s390x ppc64le
```

Comment:
For pushing, this is set here in Makefile, but for the actual builds, this is set in Travis. Could there be one source for this? Or at least write comments in both places that link the two.

Reply:
good point, I cannot have just one env variable to specify it, but definitely will add information about linkage

Reply:
moved to travis.yaml